docs/communication_protocols/dronecan_protocol.rst

```diff
@@ -162,6 +162,12 @@ This example requires an SLCAN device such as the `Zubax Babel <https://zubax.co
 software that can emulate COM ports. In this example, we are using `com0com <https://com0com.sourceforge.net/>`_. Please note that the following example will only work on the Windows operating 
 system, though a similar approach can be taken on Linux/UNIX.
 
+.. note::
+	If com0com does not produce a numbered virtual COM port or if there is a warning in the device manager for the com0com serial port emulator, it is recommended to update your drivers 
```

Comment:
Can we say, "... produce a numbered virtual COM port (as shown below), or if there ..."